.travis.yml

```diff
@@ -1,9 +1,8 @@
 sudo: required
 dist: trusty
 rvm:
-  - 2.2.7
```

Comment:
Dropped ruby 2.2 run because the latest version of webdriver is not supported. Could pin webdriver to an older version if this isn't that seems better. 

https://travis-ci.org/cheezy/page-object/jobs/570236238

Reply:
Ruby 2.2 EOL was March 2018; Ruby 2.3 EOL was March 2019. Selenium 4 will remove support for Ruby 2.3, so running on 2.5 & 2.6 is better than 2.2 & 2.3